Orange/base.py

```diff
@@ -179,6 +184,9 @@ def active_preprocessors(self):
                 self.preprocessors is not type(self).preprocessors):
             yield from type(self).preprocessors
 
+    def fitted_parameters(self, *args, **kwargs) -> List:
```

Comment:
`List` -> `list[FittedParameter]` (we can start using lowercase list from since we've dropped support for Python 3.8).